.circleci/src/commands/notify.yml

```diff
@@ -0,0 +1,79 @@
+description: Send a notification (to Mattermost) at the end of a job, based on success or failure. Must be the last step in a job.
+parameters:
+  success_message:
+    default: ''
+    type: string
+  failure_mentions:
+    default: '@max.ammann @stefanie.metzger @sarah.sporck @steffen.kleinle'
```

Comment:
i think steffen added me recently to integreat app, i think it would make sense here, too